scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+        if (i % 3 == 0):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+        elif (i % 3 == 1):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=500),
+                timezone.now() - timedelta(days=100),
+                host_team
+            )
+        elif (i % 3 == 2):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() + timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+
+
+def create_challenge_object(title, start_date, end_date, host_team):
+    fake = Factory.create()
+    evaluation_script = open(os.path.join(settings.BASE_DIR, 'examples', 'example1', 'string_matching.zip'), 'rb')
+    Challenge.objects.create(
+        title=title,
+        short_description=fake.paragraph(),
+        description=fake.paragraph(),
+        terms_and_conditions=fake.paragraph(),
+        submission_guidelines=fake.paragraph(),
+        evaluation_details=fake.paragraph(),
+        evaluation_script=SimpleUploadedFile(evaluation_script.name, evaluation_script.read()),
+        creator=host_team,
+        published=True,
+        enable_forum=True,
+        anonymous_leaderboard=False,
+        start_date=start_date,
+        end_date=end_date,
+    )
+    print "Challenge created with \n title: %s \n creator: %s \n start_date: %s \n end_date: %s"\
+          % (title, host_team.team_name, start_date, end_date)
+
+
+def create_challenge_phases(number_of_phases=1):
+    fake = Factory.create()
+    for i in range(number_of_phases):
+        name = "%s Phase" % (fake.first_name())
+        ChallengePhase.objects.create(
+            name=name,
+            description=fake.paragraph(),
+            leaderboard_public=True,
+            is_public=True,
+            start_date=challenge.start_date,
+            end_date=challenge.end_date,
+            challenge=challenge,
+            test_annotation=SimpleUploadedFile(fake.file_name(extension="txt"),
+                                               "1\n2\n3\n4\n5\n6\n7\n8\n9\n10", content_type="text/plain"),
+            codename="%s%d" % ("phase", i+1),
+        )
+        print "Challenge Phase created with \n name: %s \n challenge: %s" % (name, challenge.title)
+
+
+def create_leaderboard():
+    schema = {
+        'labels': ['score', ],
+        'default_order_by': 'score',
+    }
+    leaderboard = Leaderboard.objects.create(
+        schema=schema
+    )
+    print "Leaderboard created"
+    return leaderboard
+
+
+def create_dataset_splits(number_of_splits):
+    for i in range(number_of_splits):
+        global DATASET_SPLIT_ITERATOR
```

Comment:
I am actually not able to understand what is the reason behind for using global here. I guess if you would have done simply DATASET_SPLIT_ITERATOR +=1, it would have incremented global `DATASET_SPLIT_ITERATOR` value. Anything that I am missing?